editor.planx.uk/package.json

```diff
@@ -57,6 +57,7 @@
     "nanoid-good": "^3.1.0",
     "natsort": "^2.0.3",
     "navi": "^0.15.0",
+    "ot-json0": "^1.1.0",
```

Comment:
Good score here despite not being updated in while - it actually has zero dependencies - looks good! https://socket.dev/npm/package/ot-json0